packages/lwc-language-server/src/javascript/compiler.ts

```diff
@@ -91,7 +93,16 @@ export async function compileSource(source: string, fileName: string = 'foo.js')
             namespace: 'x',
             files: {},
         };
-        const transformerResult = await transform(source, fileName, options);
+
+        let codeTransformed = source;
+
+        if (fileName.endsWith('.ts')) {
+            codeTransformed = transformTypescript(fileName, source);
```

Comment:
I don't think this is a good idea. In case of errors we wouldn't be able to provide accurate information that maps to the line & column of the component file(s).

Reply:
Good catch - I think applying this should fix it.


https://github.com/salesforce/lwc-test/pull/75

Reply:
Is there a reason why lwc compiler cannot support `.ts`?

Reply:
Well, good question to ask the LWC team. As of now it's not.